cookbook/recipes/agent.rb

```diff
@@ -9,16 +9,15 @@
 end
 # rubocop:enable Metrics/LineLength
 
-baragon_agent_jar = "BaragonAgentService-#{node[:baragon][:version]}.jar"
-
-remote_file "/usr/share/java/#{baragon_agent_jar}" do
-  action   :create
-  backup   5
-  owner    'root'
-  group    'root'
-  mode     0644
-  source   "file://#{Chef::Config[:file_cache_path]}/Baragon/" \
-           "BaragonAgentService/target/#{baragon_agent_jar}"
+remote_file "/usr/share/java/BaragonAgentService-#{node[:baragon][:version]}-shaded.jar" do
```

Comment:
If we are including a maven install option for `BaragonService`, I think we should add the same for `BaragonAgentService` as well.